### PR TITLE
[features/AEKR-2093] updating scripts for add indexes-constraints, update vocab, update custom concepts

### PR DESCRIPTION
**Pull request by silviajimeneznavarro-edence**

### CODE REVIEW

File: local-installation/separate-scripts/start-add-indexes-and-constraints.cmd
Comment (petermoorthamer) [inline local-installation/separate-scripts/start-add-indexes-and-constraints.cmd:1]: Let's rename this script to 'add-omop-indexes-and-constraints.cmd'
The existing script 'start-omop-indexes-and-constraints.cmd' can be removed

File: local-installation/separate-scripts/start-add-indexes-and-constraints.sh
Comment (petermoorthamer) [inline local-installation/separate-scripts/start-add-indexes-and-constraints.sh:1]: Let's rename this script to 'add-omop-indexes-and-constraints.sh'
The existing script 'start-omop-indexes-and-constraints.sh' can be removed

**Review by petermoorthamer — CHANGES_REQUESTED** [review]

File: local-installation/separate-scripts/start-add-indexes-and-constraints.cmd
Comment (silviajimeneznavarro-edence) [reply local-installation/separate-scripts/start-add-indexes-and-constraints.cmd:1]: done!

File: local-installation/separate-scripts/start-add-indexes-and-constraints.sh
Comment (silviajimeneznavarro-edence) [reply local-installation/separate-scripts/start-add-indexes-and-constraints.sh:1]: done!

**Review by petermoorthamer — APPROVED** [review]

**Pull request merged**